nuxeo-services/nuxeo-platform-directory/nuxeo-platform-directory-api/src/main/resources/OSGI-INF/pageproviders-contrib.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.platform.directory.pageproviders">
+
+    <extension target="org.nuxeo.ecm.platform.query.api.PageProviderService"
+               point="providers">
```

Comment:
The indent in XML files is 2 characters (https://doc.nuxeo.com/corg/xml-or-xml-like-files-code-style/).